service/controller/resource/promxy/app/create.go

```diff
@@ -0,0 +1,36 @@
+package app
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/prometheus-meta-operator/service/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	objectMeta, err := r.getObjectMeta(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("checking if %s app exists", objectMeta.Name))
+	app, err := r.k8sClient.G8sClient().ApplicationV1alpha1().Apps(objectMeta.Namespace).Get(ctx, objectMeta.Name, metav1.GetOptions{})
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	app.Spec.Config.ConfigMap.Name = key.PromxyConfigMapName()
+	app.Spec.Config.ConfigMap.Namespace = key.PromxyConfigMapNamespace()
```

Comment:
Feels wrong to change this here, I'd rather have this being changed in the promxy app itself, or being a default in installation but not here. Again this go in the direction of my previous comment.

> I think we should not create the promxy configmap. It should be a guarantee that promxy configmap is present.
>
> That's the only interface we should agree on: having a configmap in a given namespace with a given name.

Reply:
I agree, I wanted to get this out first and I do not know how to configure architect to create a CM to do it.